### ISSUE TITLE
BATCH NORM pytorch on Xavier's experiment

### ISSUE THREAD

**Issue by brando90** [brando90/overparametrized_experiments-#23]
https://discuss.pytorch.org/t/example-on-how-to-use-batch-norm/216
